src/energyplus/CMakeLists.txt

```diff
@@ -605,9 +605,14 @@ set(${target_name}_test_src
   Test/AirLoopHVACUnitarySystem_GTest.cpp
   Test/AirLoopHVACUnitaryHeatCoolVAVChangeoverBypass_GTest.cpp
   Test/AirTerminalDualDuctConstantVolume_GTest.cpp
+  Test/AirTerminalDualDuctVAV_GTest.cpp
   Test/AirTerminalDualDuctVAVOutdoorAir_GTest.cpp
   Test/AirTerminalSingleDuctConstantVolumeFourPipeBeam_GTest.cpp
   Test/AirTerminalSingleDuctInletSideMixer_GTest.cpp
+  Test/AirTerminalSingleDuctVAVNoReheat_Gtest.cpp
+  Test/AirTerminalSingleDuctVAVReheat_Gtest.cpp
+  Test/AirTerminalSingleDuctVAVHeatAndCoolNoReheat_Gtest.cpp
+  Test/AirTerminalSingleDuctVAVHeatAndCoolReheat_Gtest.cpp
```

Comment:
@joseph-robertson be careful: windows isn't case sensitive but Ubuntu is: `Gtest` != `GTest`

```diff
-  Test/AirTerminalSingleDuctVAVHeatAndCoolNoReheat_Gtest.cpp
+  Test/AirTerminalSingleDuctVAVHeatAndCoolNoReheat_GTest.cpp
```

This is why the Ubuntu runners are failing at CMake configure: https://ci.commercialbuildings.dev/blue/organizations/jenkins/openstudio-incremental-ubuntu-18-04/detail/PR-4381/18/pipeline#step-37-log-253

Reply:
Addressed in aa348bb1dd54efb4ebdba0ebdf72fa02fc0b0871